.circleci/config.yml

```diff
@@ -106,6 +106,12 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - build-examples-splunkhecexporter:
```

Comment:
Can we avoid having component specific entries in CircleCI? It is best to add this to main makefile as a target that iterates over components and executes corresponding targets in component makefiles if they exists.

Reply:
Tigran, you assume I am a competent developer. While I believe I understood your comment, I might not be best placed to perform this change. I'll try though.

Reply:
I have changed the makefile - just not sure how to iterate over components.

Reply:
This is good enough, we can worry about it when we have more examples.